README.md

```diff
@@ -2,12 +2,22 @@
 
 Hazelcast Cluster lifecycle manager for native client tests. Server container and language clients provide easy management of server side from clients
 
+### Cluster methods running with server jars
+
 * Create a cluster with a provided xml config
 * Start a Member in the configured cluster
 * Shutdown a member
 * Terminate a member
 * Remote script execution on the container
 
+### Cloud API usage
```

Comment:
Can you also document or point to a document for the REST http requests and responses for each of these functions?

Reply:
Cloud API referance documentation is added to readme